### PR TITLE
Fix code quality and split off network validator in favor of reduced complexity

### LINKED ISSUE #33: Fix "CleanCode/ElseExpression" issue in src/HylianShield/Validator/Range.php
The method initialize uses an else expression. Else is never necessary and you can simplify the code to work without else.

https://codeclimate.com/github/HylianShield/validator/src/HylianShield/Validator/Range.php#issue_56156cdc29794d00012c7df4

### PR DESCRIPTION
**Pull request by johmanx10**
closes #33 #24 #23  #25 #26 #27

### CODE REVIEW

File: src/HylianShield/Validator/Url/Network/InstructionSet.php [hunk 1]
```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * An instruction set for network validators.
```
Comment (CyberSecutor) [inline src/HylianShield/Validator/Url/Network/InstructionSet.php]: Please add some extra information about the intention of the class. Makes it easier to understand for others.

File: src/HylianShield/Validator/Url/Network/ProtocolDefinitionInterface.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    /**
+     * Getter for the invalidParameters property.
+     *
+     * @return array
+     * @throws \LogicException when property invalidParameters is not set.
+     */
+    public function getInvalidParameters();
+
+    /**
+     * Whether the definition holds invalid parameters.
```
Comment (CyberSecutor) [inline src/HylianShield/Validator/Url/Network/ProtocolDefinitionInterface.php]: How can a definition hold invalid parameters?

File: src/HylianShield/Validator/Url/Network/InstructionSet.php (hunk 1 above)
Comment (johmanx10) [inline src/HylianShield/Validator/Url/Network/InstructionSet.php]: This comment has been resolved in the lines directly below this one.

File: test/HylianShield/Tests/Validator/LogicalGateTestBase.php [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 namespace HylianShield\Tests\Validator;
 
+use HylianShield\ValidatorInterface;
```
Comment (CyberSecutor) [inline test/HylianShield/Tests/Validator/LogicalGateTestBase.php]: Slash

**Pull request merged**